client.go

```diff
@@ -130,3 +136,13 @@ func (c Client) Request(method, endpoint string, data, response interface{}) err
 	// Done!
 	return nil
 }
+
+var netClient = &http.Client{
```

Comment:
If we're making the default client "optional", I'd rather not having the global var with it already instantiated.

Reply:
Oh yeah, excellent point!